file_util/src/file.cc

```diff
@@ -0,0 +1,140 @@
+#include <leatherman/file_util/file.hpp>
+#include <boost/nowide/fstream.hpp>
+#include <boost/nowide/cenv.hpp>
+#include <boost/system/error_code.hpp>
+#include <sstream>
+#include <leatherman/logging/logging.hpp>
+
+#include <wordexp.h>
```

Comment:
@Magisus We have replaced shellExpand with https://github.com/puppetlabs/pegasus/blob/master/lib/src/file_utils.cc#L30 in Pegasus which doesn't rely on wordexp.h @parisiale This is all we need in the agent too, right?